internal/api/v1beta1/job_types_test.go

```diff
@@ -0,0 +1,72 @@
+package v1beta1
```

Comment:
Added these tests just to keep coverage up. Coverage dipped in a couple packages.